resources/leiningen/new/xiana/project.clj

```diff
@@ -44,6 +46,6 @@
                        :migration-dir "migrations"
                        :db            {:classname   "com.mysql.jdbc.Driver"
                                        :subprotocol "postgres"
-                                       :subname     "//localhost/controllers"
+                                       :subname     "//localhost:5433/{{sanitized-name}}"
```

Comment:
Is it possible to load the config for lein migrate from config.edn?
There is >2y unresolved issue on that here https://github.com/yogthos/migratus-lein/issues/24

Reply:
It's only for migratus plugin `lein migratus migrate`